source/slang/slang-ir-inst-defs.h

```diff
@@ -594,6 +594,7 @@ INST_RANGE(TerminatorInst, Return, Unreachable)
 
 INST(RequirePrelude, RequirePrelude, 1, 0)
 INST(RequireGLSLExtension, RequireGLSLExtension, 1, 0)
+INST(RequireComputeDerivative, RequireComputeDerivative, 0, 0)
```

Comment:
Maybe this should be a HOISTABLE inst so that we will have one instance of it in the global scope. Then we can simply use rhe existing reference graph to find referencingEntryPoints.

Reply:
If this is for C-like-targets, we don't currently build a entry-point reference graph, I changed the logic to build one.

If this is just about not emitting multiple instructions for `RequireComputeDerivative`, I believe we still need to associate an instruction<->function in case we have multiple entry points (some use, some do not compute derivatives).

Reply:
Well, once it is hoistable, we can simply check all entry points referencing the only instance of the requireDerivative inst, like how we are handling all the other builtin variables.

Reply:
Ah, I see what you mean, we still need to have something in the function body that references the global thing so this isn't going to do much.